Frends.AzureDataLake.UploadFiles/Frends.AzureDataLake.UploadFiles/Definitions/Destination.cs

```diff
@@ -0,0 +1,70 @@
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace Frends.AzureDataLake.UploadFiles.Definitions;
+
+/// <summary>
+/// Destination parameters.
+/// </summary>
+public class Destination
+{
+    /// <summary>
+    /// Connection method to be used to connect to Azure Data Lake.
+    /// </summary>
+    /// <example>ConnectionMethod.ConnectionString</example>
+    [DefaultValue(ConnectionMethod.ConnectionString)]
+    public ConnectionMethod ConnectionMethod { get; set; }
+
+    /// <summary>
+    /// Name of the Azure Data Lake container.
+    /// Task will convert all letters to lowercase.
+    /// </summary>
+    /// <example>examplecontainer</example>
+    [DisplayFormat(DataFormatString = "Text")]
+    public string ContainerName { get; set; }
+
+    /// <summary>
+    /// Connection string for Azure Data Lake.
+    /// </summary>
+    /// <example>DefaultEndpointsProtocol=https;AccountName=accountname;AccountKey=Pdlrxyz==;EndpointSuffix=core.windows.net</example>
+    [UIHint(nameof(ConnectionMethod), "", ConnectionMethod.ConnectionString)]
+    [PasswordPropertyText]
+    [DisplayFormat(DataFormatString = "Text")]
+    public string ConnectionString { get; set; }
+
+    /// <summary>
+    /// Name of the Azure storage account.
+    /// </summary>
+    /// <example>Storager</example>
+    [UIHint(nameof(ConnectionMethod), "", ConnectionMethod.OAuth2)]
+    public string StorageAccountName { get; set; }
```

Comment:
Shouln't all those string properties have also the DisplayFormat attributes? (including tenant and app ids as well)

I think that it goes to "Text" mode by default if I remember correctly. So maybe either just delete this DisplayFormat from other fields or add them to all the string fields as well. Just for consistency's sake :)

Reply:
yeah, I'll delete them, I've just copied from BlobStorage solution, but I see now that there is a lot of messy code that shouldn't be populated :(